### PR TITLE
[AAE-9137] Fix imported project version

### PR DESCRIPTION
**Pull request by RahulKumar1606**
Process version is showing 0.0.2 when importing a project. 
No matter what was the process version before, it will be 0.0.2 after import.

https://github.com/Activiti/Activiti/issues/4348

### CODE REVIEW

**Comment by matteo4diani** [comment]
The business logic looks good 👍🏼 Can you add unit tests and, if it's an easy win, also integration tests? Try to add them to pre-existing test classes to simplify your work 🙏🏼

**Review by matteo4diani — CHANGES_REQUESTED** [review]
The business logic looks good 👍🏼 Can you add unit tests and, if it's an easy win, also integration tests? Try to add them to pre-existing test classes to simplify your work 🙏🏼

**Review by miguelruizdev — CHANGES_REQUESTED** [review]
Code looks alright. 
The only strong opinion on my end: we need tests, especially efficient unit ones.
Try to always come up with a reproducer for the bug/issue that you are trying to fix, and from there try to make it green by adding changes to production code. Check this for a more comprehensive description of what TDD is: https://www.browserstack.com/guide/what-is-test-driven-development#:~:text=In%20layman's%20terms%2C%20Test%20Driven,unit%20test%20creation%2C%20and%20refactoring.

Some requests on the pull request itself:
* Try to add a concise description of your changes when you open your draft/PR (a concise summary of the ticket will do just fine)
* Check [this guide](https://alfresco.atlassian.net/wiki/spaces/AP/pages/470909/How+to+commit+and+create+a+PR+in+Activiti) on how to proceed when committing on community projects (the community outside does not have access to our private Jira, so we need to referece a duplicated issue on the community repos that you have to create.
* Make sure that your commits are signed, otherwise you will not be able to merge due to security checks. You can follow [this guide](https://alfresco.atlassian.net/wiki/spaces/GUIL/pages/925402128/Signing+commits)

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/test/java/org/activiti/cloud/services/modeling/service/ProjectServiceImplTest.java [hunk 1]
```diff
@@ -18,15 +18,9 @@
...
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (matteo4diani) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/test/java/org/activiti/cloud/services/modeling/service/ProjectServiceImplTest.java]: Revert all the wildcard imports and check your IDE settings, when optimising imports IntelliJ must default to single class import -> https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

**Review by matteo4diani — CHANGES_REQUESTED** [review]

**Review by matteo4diani — APPROVED** [review]
LGTM 👍🏼 Great job @RahulKumar1606

**Review by miguelruizdev — APPROVED** [review]
LGTM! congrats on your first PR!

**Pull request merged**